ts/editor/editor-toolbar/TextColorButton.svelte

```diff
@@ -126,31 +128,37 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 </script>
 
 <WithColorHelper {color} let:colorHelperIcon let:setColor>
-    <IconButton
-        tooltip="{tr.editingTextColor()} ({getPlatformString(setCombination)})"
-        {disabled}
-        on:click={setTextColor}
-        --border-left-radius="5px"
-    >
-        {@html textColorIcon}
-        {@html colorHelperIcon}
-    </IconButton>
-    <Shortcut keyCombination={setCombination} on:action={setTextColor} />
-
-    <IconButton
-        tooltip="{tr.editingChangeColor()} ({getPlatformString(pickCombination)})"
-        {disabled}
-        widthMultiplier={0.5}
-        iconSize={120}
-    >
-        {@html chevronDown}
-        <ColorPicker
-            keyCombination={pickCombination}
-            on:input={(event) => {
-                color = setColor(event);
-                bridgeCommand(`lastTextColor:${color}`);
-            }}
-            on:change={() => setTextColor()}
-        />
-    </IconButton>
+    <ButtonGroup>
+        <ButtonGroupItem>
```

Comment:
For a similar reason, I'd like to keep the `ButtonGroup` out of the button components. The name does not imply that it's a button group, and more importantly, the text color button itself doesn't need the ButtonGroup to function, so the user should wrap it in a button group instead (or not).

Reply:
Should I resort to hard-coded radii instead? `ButtonGroup` handles the radius of its items nicely, it would be a pity to lose this functionality.

Reply:
See #2069